rosdep/base.yaml

```
@@ -2285,7 +2285,11 @@ libcoin80-dev:
     jessie: [libcoin80-dev]
     stretch: [libcoin80-dev]
   fedora: [Coin2-devel]
-  ubuntu: [libcoin80-dev]
+  ubuntu:
+    xenial: [libcoin80-dev]
+    bionic: [libcoin80-dev]
+    focal: [libcoin-dev]
+    groovy: [libcoin-dev]
```

I think to be future-proof, it can just be
```suggestion
    '*': [libcoin-dev]
    xenial: [libcoin80-dev]
    bionic: [libcoin80-dev]
```

---

Thanks, changes done as suggested.